packages/react-playground/src/App.js

```diff
@@ -37,6 +45,10 @@ class App extends React.Component {
     this.setState({ disabled: !this.state.disabled });
   }
 
+  setSize(event) {
+    this.setState({ size: this.buttonSizes[event.detail.selectedIndex].value });
```

Comment:
Both current. This line of code doesn't do anything at the moment. Perhaps best left out until I get to #46.